mungegithub/github/github.go

```diff
@@ -1578,13 +1578,17 @@ func (obj *MungeObject) UnassignPR(assignees ...string) error {
 func (obj *MungeObject) AssignPR(owner string) error {
 	config := obj.config
 	prNum := *obj.Issue.Number
-	assignee := &github.IssueRequest{Assignee: &owner}
+	assigneeList := []string{owner}
+	for ix := range obj.Issue.Assignees {
+		assigneeList = append(assigneeList, *obj.Issue.Assignees[ix].Login)
+	}
+	assignees := &github.IssueRequest{Assignees: &assigneeList}
 	config.analytics.AssignPR.Call(config, nil)
 	glog.Infof("Assigning PR# %d  to %v", prNum, owner)
 	if config.DryRun {
 		return nil
 	}
-	if _, _, err := config.client.Issues.Edit(config.Org, config.Project, prNum, assignee); err != nil {
```

Comment:
I can't believe we're doing this ...

I think we should be using the [add-assignees interface](https://developer.github.com/v3/issues/assignees/#add-assignees-to-an-issue) (implemented [here](https://github.com/google/go-github/blob/master/github/issues_assignees.go#L51)) rather than the [edit interface](https://developer.github.com/v3/issues/#edit-an-issue). 

This has way too many chances to reach a race-condition (a loop can take up to 20 minutes, and we fetch the issues at the beginning, so if anybody changes the assignees list in that time, it will be gross).

Note that this is of course no worse than before, so we can probably fix that in a following PR